src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -491,5 +547,33 @@ public void closeReaderGroup() {
             }
         }
 
+        /**
+         * Chunks the list of segment ranges between the number consumers.  If the number of segment ranges is
+         * divisible by the number of consumers then each consumer will recieve an equal number of segments, otherwise
+         * some consumers may receive more segments than others.
+         *
+         * @return A list of lists, each list representing the segments assigned to that consumer
+         */
+        private List<List<SegmentRange>> assignSegmentsToConsumers(List<SegmentRange> segmentRanges, int consumers) {
```

Comment:
Consider using `com.google.common.collect.Lists.partition` instead of this.

Reply:
Originally I did use Google `Lists.partition` but the problem with that API is that partitions the list based on a specific partition size and NOT (as I'd assumed) by the number of required partitions.  

This lead to getting into calculating the required size of partitions along with handling the cases where the number of SegmentRanges wasn't completely divisible by the number of consumers.  In the end it felt a lot simpler to partition the list as done here.